channels/urbdrc/client/libusb/libusb_udevice.c

```diff
@@ -290,11 +290,7 @@ static void LIBUSB_CALL func_iso_callback(struct libusb_transfer* transfer)
 			/* fallthrough */
 			WINPR_FALLTHROUGH
 		case LIBUSB_TRANSFER_CANCELLED:
-			/* fallthrough */
-			WINPR_FALLTHROUGH
 		case LIBUSB_TRANSFER_TIMED_OUT:
-			/* fallthrough */
-			WINPR_FALLTHROUGH
```

Comment:
accidental change?

Reply:
Nope it's a gcc warning I have from a long time (a real fall through is when you have code in the case statement).

Reply:
then please fix the macro next time ;)
already done with https://github.com/FreeRDP/FreeRDP/pull/10541 so this commit should be dropped.